app/src/main/AndroidManifest.xml

```diff
@@ -31,7 +31,10 @@
         <activity android:name=".view.activities.RequestDetailActivity" />
         <activity android:name=".view.activities.MemberProfileActivity" />
         <activity android:name=".view.activities.SendRequestActivity" />
-        <activity android:name=".view.activities.SettingsActivity"/>
+        <activity android:name=".view.activities.SettingsActivity"
+                  android:parentActivityName=".view.activities.SettingsActivity"/>
+        <activity
+            android:name=".view.activities.AboutActivity" />
```

Comment:
You're providing the up navigation from the About activity to the Settings activity, so should not the parentActivity tag be present on AboutActivity instead of SettingsActivity?

Reply:
Oh sorry this is a blunder. I will fix it.